modules/postgres/alloydb/0.2/facets.yaml

```
@@ -4,6 +4,129 @@ version: '0.2'
+      minLength: 1
+      maxLength: 63
+      pattern: "^[a-zA-Z][a-zA-Z0-9_]*$"
+      x-ui-override-disable: true
+      x-ui-error-message: "Database name must start with a letter and contain only letters, numbers, and underscores."
+      x-ui-placeholder: "e.g., ['myapp', 'analytics']"
+      default: []
+    db_schemas:
+      type: object
+      title: Database Schemas
+      description: Schema definitions for the databases.
+      type: array
```

YAMLlint (1.37.1)

[error] duplication of key "type" in mapping

(key-duplicates)